types/recordrtc/index.d.ts

```diff
@@ -209,7 +209,7 @@ declare namespace RecordRTC {
         stopRecording(): Promise<string>;
         pauseRecording(): Promise<void>;
         resumeRecording(): Promise<void>;
-        getDataURL(): Promise<string>;
+        getDataURL(cb: (dataURL: string) => void): Promise<void>;
```

Comment:
hmm. It looks like the docs are wrong for `RecordRTCPromisesHandler.prototype.getDataURL`. Compare the [actual code](https://recordrtc.org/RecordRTC.js.html#line5800) with the "example code" a few lines above... 

So I think this line was already correct (but thanks for fixing the other case, that was definitely wrong! - [link to source code](https://github.com/muaz-khan/RecordRTC/blob/7b1c360/dev/RecordRTC.js#L234-L281))

```suggestion
        getDataURL(): Promise<string>;
```